macos/FluentUI/Core/ColorProviding.swift

```diff
@@ -0,0 +1,23 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+
+// MARK: ColorProviding
+
+/// Protocol through which consumers can provide colors to "theme" their experiences
+/// The view associated with the passed in theme will display the set colors to allow apps to provide different experiences per each view
+@objc(MSFColorProviding)
+public protocol ColorProviding {
+    /// If this protocol is not conformed to, communicationBlue variants will be used
+    @objc var primary: NSColor { get }
```

Comment:
might be me not understanding GitHub PR experience, but is this a mix of tabs and spaces?

Reply:
No. Not sure why it did that. Fixing.